src/js/map/popup.js

```diff
@@ -1,14 +1,15 @@
-import {numFmt, percFmt, Observable, setPopupStyle} from '../utils';
+import {numFmt, percFmt, Observable, setPopupStyle, formatNumericalValue} from '../utils';
```

Comment:
`numFmt` and `percFumt` are not used anymore and can be deleted.

Reply:
removed 👍